deploy-manage/tools/snapshot-and-restore.md

```diff
@@ -152,16 +148,16 @@ You can’t restore a snapshot to an earlier version of {{es}}. For example, you
 
 ### Index compatibility
 
-Any index you restore from a snapshot must also be compatible with the current cluster’s version. If you try to restore an index created in an incompatible version, the restore attempt will fail.
+Any index you restore from a snapshot must also be compatible with the current cluster’s version. If you try to restore an index created in an incompatible version, the restore attempt fails.
 
-| Index creation version | 6.8 | 7.0–7.1 | 7.2–7.17 | 8.0–8.2 | 8.3–8.18 | 9.0 |
-|------------------------|-----|---------|---------|---------|---------|-----|
-| 5.0–5.6               | ✅   | ❌       | ❌       | ❌       | ✅ ^1^   | ✅ ^1^ |
-| 6.0–6.7               | ✅   | ✅       | ✅       | ❌       | ✅ ^1^   | ✅ ^1^ |
-| 6.8                   | ✅   | ❌       | ✅       | ❌       | ✅ ^1^   | ✅ ^1^ |
-| 7.0–7.1               | ❌   | ✅       | ✅       | ✅       | ✅       | ✅ ^1, 2^ |
-| 7.2–7.17              | ❌   | ❌       | ✅       | ✅       | ✅       | ✅ ^1, 2^ |
-| 8.0–8.18              | ❌   | ❌       | ❌       | ✅       | ✅       | ✅ |
+| Index creation version | {{version.stack.base}}–{{version.stack}} | 8.3–8.19 | 8.0–8.2 | 7.2–7.17 | 7.0–7.1 | 6.8 | 
+|-----------------------| ----------------------------------------- |----------|---------|----------|---------|-----|
+| 5.0–5.6               | ✅ ^1^                                    | ✅ ^1^   | ❌      | ❌       | ❌      | ✅  |         
+| 6.0–6.7               | ✅ ^1^                                    | ✅ ^1^   | ❌      | ✅       | ✅      | ✅  | 
+| 6.8                   | ✅ ^1^                                    | ✅ ^1^   | ❌      | ✅       | ❌      | ✅  |    
+| 7.0–7.1               | ✅ ^1, 2^                                 | ✅       | ✅      | ✅       | ✅      | ❌  |       
+| 7.2–7.17              | ✅ ^1, 2^                                 | ✅       | ✅      | ✅       | ❌      | ❌  |      
+| 8.0–8.19              | ✅                                        | ✅       | ✅      | ❌       | ❌      | ❌  |   
```

Comment:
there should be a {{version.stack.base}}–{{version.stack}} row too so we can document 9-9 compatibility